### DISCUSSION TITLE
[Scam Alert] Pfxoptionmarket

### DISCUSSION THREAD

**Discussion by micro73** [Blockunity/WarningList#22] [General]
Pfxoptionmarket

ادرس سایت : https://pfxoptionmarket.com/

نوع کلاهبرداری: وعده سود های غیرمتعارف

توضیحات سایت : PFx Option Market یک شرکت آنلاین است. ما یکی از اولین کسانی بودیم که روی ارزهای دیجیتال سرمایه گذاری کردیم.
PFx Option Market یک شرکت سرمایه گذاری پیشرو در حوزه بلاک چین است.
ماموریت ما این است که به عنوان یک کاتالیزور برای پذیرش جهانی و نوآوری بلاک چین عمل کنیم.
ما فقط بر روی سرمایه گذاری در فناوری های بلاک چین تمرکز می کنیم. تیم ما هم در تامین مالی سنتی و هم در فناوری بلاک چین در حال ظهور تجربه دارد
ما معامله‌گران و مدیران پرتفوی حرفه‌ای هستیم که سود را به سرمایه‌گذاران برمی‌گردانیم که توسط کمیسیون بورس و اوراق بهادار تضمین و تنظیم می‌شود. ما خدمات معاملاتی بهینه ای را ارائه می دهیم که می تواند هر هفته سود حداکثر 18 درصد از وجوه سرمایه گذاری شده را ارائه دهد. ما بر روی طیف گسترده ای از دارایی ها، رمزارزها، گزینه های سهام و کالاها معامله می کنیم.

بررسی های انجام شده :
به محض ورود به سایت Trend Micro Check هشدار اسکم میدهد .

![11](https://user-images.githubusercontent.com/103103225/184308589-2a29033b-2745-4782-a608-cfe9daa458a6.jpg)

طبق گفته خود سایت در ارز های دیجیتال سرمایه گذاری و ترید میکنیم و سود 18 درصد انهم هفتگی به بقیه میپردازیم .
در سایت بعد از ثبت نام هیچ سند و مدرکی وجود ندارد که انها در حال چه فعالیتی هستند روی پول سرمایه گذاری میکنند یا ترید فقط صرفا شما پول واریز میکنید و سود میگیرید . (سود هفتگی 18 درصد میشود 72 درصد ماهانه = پانزی و کلاهبرداری ))

ادرس گوگل مپ فیک میباشد و سرتیفیکیت بارگزاری شده در سایت وجود خارجی ندارد.
بررسی scam adviser و اطلاعات whois

![33](https://user-images.githubusercontent.com/103103225/184308614-65e60f34-c3c3-4fab-a562-da54bfa27cbf.jpg)
![44](https://user-images.githubusercontent.com/103103225/184308618-66ba7795-7d42-4fdf-a02f-93be9c09ca07.jpg)
![55](https://user-images.githubusercontent.com/103103225/184308635-872adc99-5ce4-4841-b786-d8e45f1936d7.jpg)

ترجمه ماشینی :
مالک وب سایت با استفاده از یک سرویس پولی هویت خود را در WHOIS مخفی می کند
این وب سایت بازدید کننده زیادی ندارد
ما بسیاری از وب‌سایت‌های با رتبه پایین را در یک سرور پیدا کردیم
ما خدمات ارزهای دیجیتال را شناسایی کردیم که می توانند ریسک بالایی داشته باشند

ویژگی های یک سایت HYIP شناسایی شده است . (( در باره این مورد که بسیار شایع است و به شدت با ان برخورد میکنید در لینک زیر مطالعه کنید))

https://www.scamadviser.com/scam-reports/scam-alerts/3265/what-are-high-yield-investment-programs-hyips
روش های پرداخت ناشناس در این سایت شناسایی شد
این وب سایت به تازگی ثبت شده است.
به نظر می رسد این وب سایت از یک ارائه دهنده میزبانی با شهرت مشکوک استفاده می کند.

بسیاری از پروژه های اسکم و کلاهبرداری از شرکتی با اطلاعات زیر برای ثبت دامنه و مخفی کردن اطلاعات استفاده میکنند -
Address
Kalkofnsvegur 2
City
Reykjavik
State / Province
Capital Region
Postal Code
101
Country
IS
Phone
+354.4212434

سایت سال 2022 تاسیس شده است و اطلاعات ثبت کننده دامین مخفی شده است.

قسمت درباره ما در سایت خالی است .

![66](https://user-images.githubusercontent.com/103103225/184308674-ae85f575-a831-4bb0-af17-ed1a1ce2042b.jpg)

هشدار اسکم و کلاهبرداری

**Comment by Blockunity** [comment]
![Screenshot 2022-08-12 150053](https://user-images.githubusercontent.com/72666334/184360359-d9525ac1-5a08-46b3-a18a-d87dfaf9b092.png)
 
 پلن سرمایه گذاری که مثل خیلی از پروژه های پانزی دیگر شامل سود قطعی هست

**Comment by Blockunity** [comment]
ترافیک وب سایت مخفی و دیتایی ارائه نشده
